pkg/config/config.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// ChaosServerConfig defines the configuration for Chaos Dashboard
+type ChaosServerConfig struct {
+	ListenHost           string `enconfig:"LISTEN_HOST" default:"0.0.0.0"`
+	ListenPort           int    `enconfig:"LISTEN_PORT" default:"2333"`
+	MetricAddress        string `enconfig:"METRIC_ADDRESS"`
+	EnableLeaderElection bool   `enconfig:"ENABLE_LEADER_ELECTION"`
+	Database             *DatabaseConfig
+}
+
+// DatabaseConfig defines the configuration for databases
+type DatabaseConfig struct {
+	// Archive  Chaos Experiments to DB.
+	Archive    bool
+	Driver     string `envconfig:"DATABASE_DRIVER"     default:"sqlite3"`
```

Comment:
Grafana didn't support sqlite3 data source. Is it okay to use it as the default?

Reply:
I think this is OK.  If users did not choose MySQL, we can't provide Grafana support.

Reply:
Make sense! But we need to mention this in our docs later